tests/rss/test_rss.py

```diff
@@ -1,21 +1,26 @@
-from autoplex.data.rss.jobs import do_rss
+import os 
+os.environ["OMP_NUM_THREADS"] = "1"
+
+from autoplex.data.rss.jobs import do_rss_single_node, do_rss_multi_node
+import shutil
+from pathlib import Path
 from jobflow import run_locally
 from ase.io import read
 from pymatgen.io.ase import AseAtomsAdaptor
 import numpy as np
 
 
-def test_gap_rss(test_dir, memory_jobstore, clean_dir):
```

Comment:
Why is `clean_dir` removed?

Reply:
Because I need to know the output during debugging to make adjustments. I'll add it back now.